usagov/vote-gov-drupal
# VOTE-854 Apply styling to the header components

Delete all details that do not apply to this PR!

Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-854

Description (optional)
Apply styling to the header region div
Apply styling to the government banner component of the header
Apply styling to the search bar
Apply styling to the language switcher
Apply styling to the site-wide alert

Testing steps
Check out the feature/VOTE-894-header-styling
Run lando retune
View the website in browser and check the following:
a. The default state of the accordion is collapsed
b. The background color is #f0f0f0
c. The images and icons are visible
d. The layout of each component within the header region is correct.
